terraform/deployments/tfc-configuration/gcp-projects.tf

```diff
@@ -0,0 +1,32 @@
+module "gcp-ga4-analytics-352613" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.10.0"
+
+  organization        = var.organization
+  workspace_name      = "gcp-ga4-analytics-352613"
+  workspace_desc      = "BigQuery infrastructure for Data Insight & Analytics"
+  workspace_tags      = ["production", "ga4-analytics-352613", "gcp"]
+  terraform_version   = var.terraform_version
+  execution_mode      = "remote"
+  working_directory   = "/terraform/deployments/ga4-analytics-352613/"
+  trigger_patterns    = ["/terraform/deployments/ga4-analytics-352613/**/*"]
+  global_remote_state = true
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
+    "GOV.UK Non-Production" = "write"
+    "GOV.UK Production"     = "write"
+  }
+
+  variable_set_names = [
+    "gcp-credentials-production",
+    "common",
+    "common-integration"
```

Comment:
You may not actually need the common var sets on this. Review them remove them from the list if they aren't necessary

https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/tfc-configuration/variables-common.tf

https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/tfc-configuration/variables-integration.tf

Reply:
Thanks for flagging, i'll remove the common ones because they aren't used.